src/org/zaproxy/zap/extension/tokengen/TokenGenMessagesTableModel.java

```diff
@@ -1,10 +1,10 @@
 /*
  * Zed Attack Proxy (ZAP) and its related class files.
- * 
+ *
```

Comment:
Does that mean that you will leave the indentation changes? Would you mind reverting the changes at least in this file? (which no longer has any code changes)